metricbeat/module/windows/services/services_windows.go

```diff
@@ -0,0 +1,370 @@
+// +build windows
+
+package services
+
+import (
+	"bytes"
+	"strconv"
+	"syscall"
+	"unicode/utf16"
+	"unsafe"
+
+	"github.com/elastic/beats/libbeat/common"
```

Comment:
All `github.com/elastic` imports should be in the third group (it goes stdlib, third-party, elastic).